packages/access-client/src/agent.js

```diff
@@ -21,13 +23,50 @@ import {
   validate,
   canDelegateCapability,
 } from './delegations.js'
-import { AgentData } from './agent-data.js'
+import { AgentData, getSessionProofs } from './agent-data.js'
+import { createDidMailtoFromEmail } from './utils/did-mailto.js'
+import { addProvider, waitForDelegationOnSocket } from './agent-use-cases.js'
 
-export { AgentData }
+export { AgentData, createDidMailtoFromEmail }
+export * from './agent-use-cases.js'
 
 const HOST = 'https://access.web3.storage'
 const PRINCIPAL = DID.parse('did:web:web3.storage')
 
+/** @type {WeakMap<Agent, AgentData>} */
+export const agentToData = new WeakMap()
```

Comment:
Explain why to use this.
Explain long term plan.
Avoid exporting it if possible.